clients/bigquery/tableid_test.go

```diff
@@ -23,3 +23,11 @@ func TestTableIdentifier_FullyQualifiedName(t *testing.T) {
 	// Table name that is a reserved word:
 	assert.Equal(t, "`project`.`dataset`.`table`", NewTableIdentifier("project", "dataset", "table").FullyQualifiedName())
 }
+
+func TestTableIdentifier_EscapedTable(t *testing.T) {
+	// Table name that is not a reserved word:
```

Comment:
I don't think we need to test for reserved words anymore since we quote everything.